tgui/packages/tgui/interfaces/LocalGenerator/index.tsx

```diff
@@ -0,0 +1,186 @@
+
+import { useBackend } from '../../backend';
+import { Window } from '../../layouts';
+import { Section, Stack, Button, Box, ProgressBar } from '../../components';
+import { PortableHoldingTank } from '../common/PortableAtmos';
+import { LocalGeneratorData } from './type';
+
+export const LocalGenerator = (props, context) => {
+  const { act, data } = useBackend<LocalGeneratorData>(context);
+
+  const {
+    name,
+    holding,
+    internalCell,
+    connectedAPC,
+    chargeAPC,
+    boltsStatus,
+    generatorStatus,
+  } = data;
+
+  const handleToggleBolts = () => {
+    act("toggle-bolts");
+  };
+
+  const handleToggleGenerator = () => {
+    act("toggle-generator");
+  };
+
+  const handleSwapChargeTarget = () => {
+    act("swap-target");
+  };
+
+  const handleEjectTank = () => {
+    act("eject-tank");
+  };
+
+  const handleEjectCell = () => {
+    act("eject-cell");
+  };
+
+  const handleConnectAPC = () => {
+    act("connect-APC");
+  };
+
+  return (
+    <Window title={name} width={300} height={470}>
+      <Window.Content>
+        <Stack vertical>
+          <Section title="Generator">
+            <Stack vertical>
+              <Stack.Item>
+                <Stack align="baseline">
+                  <Stack.Item basis="50%">
+                    <Button
+                      width={11}
+                      content="Toggle Floor Bolts"
```

Comment:
With the disclosure that I have *not* been keeping up with tgui lately, do `Button` components take `content` rather than simply putting the label as its children?